docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -339,27 +570,39 @@ _Triggers the **[UniversalReceiver](#universalreceiver-1)** event when this func
 
 #### Return Values:
 
-| Name     | Type    | Description                            |
-| :------- | :------ | :------------------------------------- |
-| `result` | `bytes` | Can be used to encode response values. |
+| Name     | Type    | Description                                                                    |
+| :------- | :------ | :----------------------------------------------------------------------------- |
+| `result` | `bytes` | The return value of the UniversalReceiverDelegates contracts encoded as bytes. |
 
 ### isValidSignature
 
+:::info
+
+Check the [**isValidSignature(..)**](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md#isvalidsignature) function specification in [**LSP0-ERC725Account Standard**](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md) in the **LIP repository**.
+
+Check the [**isValidSignature(..)**](https://github.com/lukso-network/lsp-smart-contracts/blob/v0.8.0/contracts/LSP0ERC725Account/LSP0ERC725AccountCore.sol#L159) function implementation in **LSP0ERC725Account** Contract.
+
+:::
+
 ```solidity
 function isValidSignature(
     bytes32 dataHash,
     bytes memory signature
 ) public view returns (bytes4 magicValue)
 ```
 
-Checks if a signature was signed by the `owner` of the contract, according to [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271). If the `owner` is a contract itself, it will call the `isValidsignature(..)` function on the owner contract, if it supports [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271). Otherwise it will return the failure value.
+Checks if a signature was signed by the `owner` of the contract, according to [**EIP-1271**](https://eips.ethereum.org/EIPS/eip-1271).
+
+If the `owner` is an EOA, the **ECDSA algorithm** will be used to recover the address of the signer from the dataHash and the signature, the function will return the magicValue pointing that the signature is valid, if the address recovered matches the address of the owner. Will return the failure value otherwise.
```

Comment:
I suggest we link to EIP1271 here.

I just put the link for `MAGIC VALUE` at the specifications section of EIP1271 as it shows it in the constants.

<img width="628" alt="image" src="https://user-images.githubusercontent.com/31145285/218110982-7b6bd26c-f8ed-464d-8586-3097a8f39e20.png">


The fail value is not explicitly stated in the EIP1271 so my suggestion would be to write `0xffffffff`

```suggestion
If the `owner` is an EOA, the **ECDSA algorithm** will be used to recover the address of the signer from the dataHash and the signature, the function will return the [**`MAGICVALUE`**](https://eips.ethereum.org/EIPS/eip-1271#specification) pointing that the signature is valid, if the address recovered matches the address of the owner. Will return the **`FAIL VALUE`** `0xffffffff` otherwise.
```